compiler/src/dmd/console.d

```diff
@@ -237,7 +237,7 @@ bool detectColorPreference() nothrow @trusted
 {
     import core.stdc.stdlib : getenv;
     const noColor = getenv("NO_COLOR");
-	return noColor == null || noColor[0] == '\0';
```

Comment:
?

Reply:
my detabber caught a tab there

Reply:
Can you set it to 4 spaces instead of 8? The indentation is wrong now